### ISSUE TITLE
View Repository redirects me to https://github.com/:user/repo

### ISSUE THREAD

**Issue by olivertzeng** [vyfor/cord.nvim#53]
### Description

The view repository button in the rpc redirects me to https://github.com/:user/repo instead of https://github.com/user/repo

### Operating system

Arch BTW

### Neovim version

```markdown
NVIM v0.10.0
Build type: Release
LuaJIT 2.1.1713773202
```

### Steps to reproduce

start connecting

### Minimal configuration

```Lua
local lazypath = vim.fn.stdpath('data') .. '/lazy/lazy.nvim'
if not (vim.uv or vim.loop).fs_stat(lazypath) then
  vim.fn.system({
    'git',
    'clone',
    '--filter=blob:none',
    'https://github.com/folke/lazy.nvim.git',
    '--branch=stable',
    lazypath,
  })
end
vim.opt.rtp:prepend(lazypath)

require('lazy').setup({
  {
    'vyfor/cord.nvim',
    build = './build',
    opts = {
      usercmds = true,
      timer = {
        enable = true,
        interval = 1500,
        reset_on_idle = false,
        reset_on_change = false,
      },
      editor = {
        image = nil,
        client = 'neovim',
        tooltip = 'The Superior Text Editor',
      },
      display = {
        show_time = true,
        show_repository = true,
        show_cursor_position = true,
        swap_fields = false,
        workspace_blacklist = {},
      },
      lsp = {
        show_problem_count = false,
        severity = 1,
        scope = 'workspace',
      },
      idle = {
        show_idle = true,
        timeout = 300000,
        disable_on_focus = true,
        text = 'Idle',
        tooltip = '💤',
      },
      text = {
        viewing = 'Viewing {}',
        editing = 'Editing {}',
        file_browser = 'Browsing files in {}',
        plugin_manager = 'Managing plugins in {}',
        workspace = 'In {}',
      },
      buttons = {
        {
          label = 'View repository',
          url = 'git',
        },
      },
    },
  },
}, {})
```

**Comment by vyfor** [comment]
Could you provide the full URL link to the repository?

**Comment by olivertzeng** [comment]
> Could you provide the full URL link to the repository?

https://github.com/olivertzeng/homework
shown as https://github.com/:olivertzeng/homework

**Comment by vyfor** [comment]
The one defined in the config, right? You could get it by running `git config --get remote.origin.url`

**Comment by olivertzeng** [comment]
> git config --get remote.origin.url

`git@github.com:olivertzeng/homework` ssh clone url
that's why

**Comment by vyfor** [comment]
Fixed in #54. Thanks!
